src/cheri-pte-ext.adoc

```diff
@@ -156,10 +156,14 @@ in xref:menvcfgreg_pte[xrefstyle=short] and xref:senvcfgreg_pte[xrefstyle=short]
 when XLEN=64. The CRG bit in *senvcfg*
 is used as described in xref:section_extending_pte[xrefstyle=short].
 
-.Machine environment configuration register (*menvcfg*) with Zcheripte, RV64 only
+WARNING: Unlike all other bits in *Xenvcfg*, CRG is in *senvcfg* and *not* in *menvcfg*. This suggests
+ this it is in the wrong place and is likely to move at some point in the future to a location
```

Comment:
Typo? It sounds a little odd.

Reply:
what's the problem?

Reply:
"This suggests this it is in the wrong place..."

Reply:
this is the point - we know it's wrong because it doesn't follow the pattern of Xenvcfg - but it's not worth changing now as all this stuff will change when we finally get to ARC review - so I think it's good enough for now. You can reword it if you like.

Reply:
This text was removed when migrating CRG to xstatus.